package.json

```diff
@@ -43,16 +43,12 @@
     "url": "http://buildit.wiprodigital.com/"
   },
   "contributors": [
-    "Gregg Anderson",
```

Comment:
Not sure why this is showing up as a change versus `develop`. We need to be careful when this PR eventually gets merged that it doesn't revert the recent update to the contributors list.

Reply:
Not sure - fixed in another commit though.